src/lamplib/src/genny/tasks/run_tests.py

```diff
@@ -163,10 +163,14 @@ def out(cmd_output: str) -> bool:
 # See the logic in _setup_resmoke.
 # These are the "Binaries" evergreen artifact URLs for mongodb-mongo compile tasks.
 # The binaries must be compatible with the version of the mongo repo checked out in use for resmoke,
-# which is the sha "cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3" mentioned below.
+# which is the sha "6e2dcc5a39eb2de9d2e8209271115c078ae16470" mentioned below.
 _canned_artifacts = {
-    "osx": "https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/macos/cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3/binaries/mongo-mongodb_mongo_master_macos_cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3_21_12_03_09_47_48.tgz",
-    "amazon2": "https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/amazon/cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3/binaries/mongo-mongodb_mongo_master_amazon_cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3_21_12_03_09_47_48.tgz",
+    "osx": "https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/macos/6e2dcc5a39eb2de9d2e8209271115c078ae16470/binaries/mongo-mongodb_mongo_master_macos_6e2dcc5a39eb2de9d2e8209271115c078ae16470_21_12_03_09_47_48.tgz",
+    "amazon2": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongo-mongodb_mongo_v6.0_amazon2_6e2dcc5a39eb2de9d2e8209271115c078ae16470_22_06_29_21_31_44.tgz",
+    "ubuntu1804": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongo-mongodb_mongo_v6.0_enterprise_ubuntu1804_64_6e2dcc5a39eb2de9d2e8209271115c078ae16470_22_06_29_21_31_44.tgz",
+    "ubuntu2004": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongo-mongodb_mongo_v6.0_ubuntu2004_6e2dcc5a39eb2de9d2e8209271115c078ae16470_22_06_29_21_31_44.tgz",
+    "rhel70": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongo-mongodb_mongo_v6.0_rhel70_6e2dcc5a39eb2de9d2e8209271115c078ae16470_22_06_29_21_31_44.tgz",
+    "rhel8": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongo-mongodb_mongo_v6.0_enterprise_rhel_80_64_bit_6e2dcc5a39eb2de9d2e8209271115c078ae16470_22_06_29_21_31_44.tgz",
```

Comment:
It's further down and unchanged, but I think the automatic search for a mongo repo and binary is happening still? I think we want to remove that logic and make sure we only pick up the mongo repo and existing binaries if users explicitly specify them with a flag.

Reply:
Yeah missed this part of the ticket.